packages/core/content-releases/admin/src/services/release.ts

```diff
@@ -187,7 +187,9 @@ const releaseApi = createApi({
       }),
       updateReleaseAction: build.mutation<
         UpdateReleaseAction.Response,
-        UpdateReleaseAction.Request
+        UpdateReleaseAction.Request & { query: GetReleaseActions.Request['query'] } & {
+          actionPath: [string, number];
```

Comment:
you don't have to change this, but incase you didn't know you can do named tuples e.g. `[string, number]` to `[actionKey: string, index: number]` :)